server/database.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @fileoverview database function handler
+ */
+var assert = require('assert');
+
+var log = require('./log.js');
+
+/** @type {database} */
+module.exports = database;
+
+/**
+ * @constructor
+ */
+function database() {}
+
+/**
+ * Checks the user information for signing in.
+ * @param {!Object} collection Database collection.
+ * @param {!Object} query Database query.
+ * @param {!Array<!Object>} data User Information.
+ * @param {function(!Object)} cursorCallback Callback function.
+ */
+database.getOne = function(collection, query, data, cursorCallback) {
+  var cursor = collection.find(query);
+  cursor.count(function(err, count) {
+    if (err) {
+      log.serverLog(err);
```

Comment:
I checked mongodb document and cannot find out the definition of error... How to get the properties of error object so that I can decide to print out which part of the error?